snuba/cli/bootstrap.py

```diff
@@ -100,11 +101,25 @@ def bootstrap(
     # tables or distributed tables, etc.
 
     # Create the tables for every dataset.
+    clickhouse_tables = ClickhousePool().execute("show tables")
+    existing_tables = [row[0] for row in clickhouse_tables]
```

Comment:
```suggestion
    existing_tables = {row[0] for row in clickhouse_tables}
```

I personally don't think there's much value in assigning `clickhouse_tables` to a local, I'd probably just inline it in the list/set comprehension.